osf_tests/test_prereg.py

```diff
@@ -21,23 +26,55 @@ def test_not_logged_in(self):
                 'has_projects': False,
                 'has_draft_registrations': False,
                 'campaign_long': 'Prereg Challenge',
-                'campaign_short': 'prereg',
+                'campaign_short': 'prereg_challenge',
                 'is_logged_in': False,
             }
         )
 
+        cache.delete(Switch._cache_key(OSF_PREREGISTRATION))
```

Comment:
Is this necessary? If you're using `override_switch` to set the value of a switch, you shouldn't have to worry about the cache.